cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -106,6 +106,15 @@ func (a *StaticAutoscaler) cleanUpIfRequired() {
 
 // RunOnce iterates over node groups and scales them up/down if necessary
 func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError {
+	if a.processors != nil && a.processors.AutoscalingStatusProcessor != nil {
```

Comment:
Can processors and a.processors.AutoscalingStatusProcessor be nil? I other executions we don't check that.